test/wanda_web/controllers/v1/checks_customization_controller_test.exs

```diff
@@ -263,7 +264,69 @@ defmodule WandaWeb.V1.ChecksCustomizationsControllerTest do
       end
     end
 
-    test "should allow customizing check values", %{
+    allowing_abilities = [
+      %{
+        name: "all abilities",
+        abilities: [
+          %{
+            name: "all",
+            resource: "all"
+          }
+        ]
+      },
+      %{
+        name: "specific abilities",
+        abilities: [
+          %{
+            name: "all",
+            resource: "check_customization"
+          }
+        ]
+      }
+    ]
+
+    for %{name: scenario_name, abilities: allowing_abilities} <- allowing_abilities do
+      @tag abilities: allowing_abilities
```

Comment:
question. does this replace the `abilities` in the context? interesting

Reply:
Yes, in `UserAwareConnCase` we either use default abilities or what has been defined in `@tag`